cmd/kube-scheduler/app/server_test.go

```diff
@@ -84,12 +84,18 @@ users:
 	// plugin config
 	pluginConfigFile := filepath.Join(tmpDir, "plugin.yaml")
 	if err := ioutil.WriteFile(pluginConfigFile, []byte(fmt.Sprintf(`
-apiVersion: kubescheduler.config.k8s.io/v1beta2
+apiVersion: kubescheduler.config.k8s.io/v1beta3
```

Comment:
v1beta2 is also getting a multiPoint field to handle round-tripping through the internal version, which is what all the logic operates on. So, this shouldn't make a difference right?

Reply:
It matters if it changes behavior of existing v1beta2 config files. Changing this here makes it hard to tell if that happened

Reply:
It shouldn't, but I split this out into a v1beta3 test and kept the old v1beta2 test.

The v1beta3 test looks different now because I specifically added MultiPoint plugins to achieve the same output as it would have been without multipoint (ie, v1beta2' s output). But the semantics for `Disabled`/`Enabled` in individual extension points is the same, and individual points take precedence, so even though default plugins are now "MultiPoint" the v1beta2 test doesn't break